components/MainNav.tsx

```diff
@@ -37,6 +37,11 @@ class MainNav extends PureComponent<MainNavProps> {
                 key={title}
                 className="text-nowrap"
                 href={path}
+                target={
+                  isXDomain(path)
```

Comment:
```suggestion
                  new URL(path, API_Host).origin  !== API_Host
```
常量原值在此：
https://github.com/kaiyuanshe/kaiyuanshe.github.io/blob/1af8e065a1bf79bf3fd333b0d8b785fcb506d9bb/models/Base.ts#L8-L12